admin/data/sql/migrations/143.do.create-check-window-with-status-view.sql

```diff
@@ -6,4 +6,4 @@ AS SELECT cw.urlSlug, cw.name, cw.isDeleted, CAST(
         ELSE 'Past'
     END AS NVARCHAR(50)
    ) AS [status]
-FROM mtc.mtc_admin.checkWindow cw
+FROM [mtc_admin].checkWindow cw
```

Comment:
👍  the database name is different across environments, and should never appear statically in the code